backend/src/apiserver/resource/resource_manager.go

```diff
@@ -199,9 +199,9 @@ func (r *ResourceManager) UnarchiveExperiment(experimentId string) error {
 	return r.experimentStore.UnarchiveExperiment(experimentId)
 }
 
-func (r *ResourceManager) ListPipelines(opts *list.Options) (
+func (r *ResourceManager) ListPipelines(filterContext *common.FilterContext, opts *list.Options) (
```

Comment:
Can we add the UT to verify that the ListPipelines method work?

Reply:
sure I added it, at least a happy path test